src/mmdata.c

```diff
@@ -2413,16 +2464,48 @@ long g_pntrStartTempAllocStack = 0;   /* Where to start freeing temporary alloca
                                     when pntrLet() is called (normally 0, except in
                                     special nested vstring functions) */
 /*!
- * \var pntrTempAllocStack
- * \brief a \ref stack "stack" of \a temp_pntrString.
+ * \var pntrString *pntrTempAllocStack[]
+ * \brief a \ref stack "stack" of \ref temp_pntrString.
  *
- * Holds pointers to temporarily allocated data of type \a pntrString.  Such a
- * \ref stack "stack" contains strictly __local__ data of a function, not
- * accessed from outer levels.
+ * Holds pointers to temporarily allocated data of type \ref temp_pntrString.  Such
+ * a \ref stack "stack" is primarily designed to operate like one for temporary
+ * allocated ad hoc operands, as described in \ref stack.  The stack top index
+ * is \ref g_pntrTempAllocStackTop, always refering to the next push position.
+ * The \ref g_pntrStartTempAllocStack supports nested operations by indicating
+ * where the operands for the upcoming operation start from.
+ * \attention A \ref pntrString consists of an array of pointers.  These
+ *   pointers may themself refer data that needs a clean up, when the last
+ *   reference  to it disappears (such as deallocating memory for example).
+ *   There is no automatic procedure handling such cases when pointers are
+ *   popped off the stack to be freed.
+ * \bug The element type should be temp_pntrString, because a NULL_PNTRSTRING
+ *   must not be pushed on the stack.
  */
 pntrString *pntrTempAllocStack[M_MAX_ALLOC_STACK];
 
-
+/*!
+ * \fn temp_pntrString *pntrTempAlloc(long size)
+ * \par size > 0
+ * allocates a \ref block capable of holding \p size \ref pntrString entries
```

Comment:
Yeah, janitory work.  I noticed it myself but did not address it because it consumes time I need elsewhere.  I know I am a beginner in doxygen, I frequently struggle with gory details like "\ref bug" that is interpreted by doxygen different to my expectations.  Then the \page s where conceptional ideas should be gathered.  The missing bird's eye view is not helpful here, so they need to be reworked more often than I like.  Won't fix for now until I better see how best practices are.  Added a \page doc-todo where I listed your observation.